indico/modules/designer/controllers.py

```diff
@@ -177,8 +179,21 @@ def _process_args(self):
 
     def _process(self):
         title = f'{self.template.title} (copy)'
-        new_template = DesignerTemplate(title=title, type=self.template.type, data=self.template.data,
-                                        **self.target_dict)
+        new_template = DesignerTemplate(title=title, type=self.template.type, **self.target_dict)
+
+        data = self.template.data
+        image_items = [(i, item) for i, item in enumerate(data['items']) if item['type'] == 'fixed_image']
+        for i, image_item in image_items:
+            if image_item.get('image_id'):
+                old_image = DesignerImageFile.get(image_item.get('image_id'))
+                new_image = DesignerImageFile(filename=old_image.filename, content_type=old_image.content_type,
+                                              template=new_template)
+                with old_image.open() as f:
+                    new_image.save(f)
+
+                data['items'][i]['image_id'] = new_image.id
+                data['items'][i]['image_url'] = new_image.download_url
```

Comment:
Why are we saving a URL in the `data`? This should be generated on the fly...

Reply:
Tbh it's because it was easier to get the url from the item in the javascript. But to change this, would it be okay to include the images url as part of the template_data?